module/applications/accordion.mjs

```diff
@@ -54,18 +54,21 @@ export default class Accordion {
     this.#sections = new Map();
     this.#ongoing = new Map();
     const { headingSelector, contentSelector } = this.#config;
+    let idx = 0;
```

Comment:
Rather than assigning an index externally, let's just convert this to a normal `for` loop instead of a `for..of`.

Reply:
The issue is that we only want to increment the index if the content is found, because otherwise an entry in `#collapsed` isn't created. If we used a `for` loop there are situations where the indexes wouldn't match.

Reply:
Ah, you're right, I missed that detail